### PR TITLE
Submission for 47022315

### PR DESCRIPTION
**Pull request by carsonalh**

### CODE REVIEW

**Comment by shaivikaaaa** [comment]
@gayanku

**Comment by yexincheng** [comment]
**This is an initial inspection, no action is required at this point**
1. **Solves problem**: No quantitative metrics were reported. No training logs or loss plots were recorded. From the presented prediction figures, it seems the model predicted nothing. Dependency requirement and reference were not written in the report.  
2. **Implementation**: The structures of functions and classes are good. But, no epochs such argument used, the training code only loop the dataset once. Learning rate and batch size are hard-coded
3. **Design**: Good
4. **Comment**: No comments at all.
5. **Difficulty**: Easy

**NOTE**:
Your project should be created under _PatternAnalysis-2024/recognition/<student_constructed_folder>_, otherwise it won't be merged
